src/main/java/it/pagopa/interop/probing/eservice/registry/updater/consumer/ServicesReceiver.java

```diff
@@ -37,36 +38,54 @@ public static ServicesReceiver getInstance() throws IOException {
   }
 
   private String sqsUrlServices;
+  private String eserviceOperationUrl;
+  private String eserviceBasePath;
 
   public ServicesReceiver() throws IOException {
     this.sqsUrlServices = PropertiesLoader.getInstance().getKey(SQS);
+    this.eserviceOperationUrl = PropertiesLoader.getInstance().getKey(ESERVICE_OPERATION_URL);
+    this.eserviceBasePath = PropertiesLoader.getInstance().getKey(ESERVICE_BASE_PATH);
   }
 
   public void receiveStringMessage() throws IOException {
 
-    try (Client client = ClientBuilder.newClient()) {
-      ObjectMapper mapper = new ObjectMapper();
-      SqsConfig sqs = SqsConfig.getInstance();
-
-      ReceiveMessageRequest receiveMessageRequest =
-          new ReceiveMessageRequest(sqsUrlServices).withMaxNumberOfMessages(10);
-      List<Message> sqsMessages =
-          sqs.getAmazonSQSAsync().receiveMessage(receiveMessageRequest).getMessages();
-
-      while (Objects.nonNull(sqsMessages) && !sqsMessages.isEmpty()) {
-        for (Message message : sqsMessages) {
-          EserviceDTO service = mapper.readValue(message.getBody(), EserviceDTO.class);
-          RestClient.getInstance().saveEservice(service, client);
-          logger.logMessageEserviceSaved(service.getEserviceId(), service.getVersionId());
-          sqs.getAmazonSQSAsync().deleteMessage(
-              new DeleteMessageRequest().withQueueUrl(sqsUrlServices)
-                  .withReceiptHandle(message.getReceiptHandle()));
-          logger.logMessageQueueMessageDeleted(service.getEserviceId(), service.getVersionId(),
-              URI.create(sqsUrlServices));
+
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Use ObjectMapper as a Singleton and not as a prototype, centralized configuration of it.